pyinstaller.spec

```diff
@@ -11,7 +11,10 @@ import struct
 import sys
 
 
-if not hasattr(sys, 'real_prefix'):
+try:
+    import allmydata
+    del allmydata
+except ImportError:
     sys.exit("Please run inside a virtualenv with Tahoe-LAFS installed.")
```

Comment:
I really have no idea what is going on here.  Maybe it has something to do with the changes to the pyinstaller tox environment?

"Can `allmydata` be imported?" seems like the more obvious spelling of "is Tahoe-LAFS installed in this environment?" than the check that was here before, though, so ... yea, great!